src/main/java/com/umc/networkingService/domain/schedule/mapper/ScheduleMapper.java

```diff
@@ -0,0 +1,101 @@
+                .title(schedule.getTitle())
+                .content(schedule.getContent())
+                .startDateTime(schedule.getStartDateTime())
+                .endDateTime(schedule.getEndDateTime())
+                .placeSetting(schedule.getPlaceSetting())
+                .hostType(schedule.getHostType())
+                .build();
+    }
+
+    public ScheduleInfoSummaryInCalendar toScheduleInfoSummaryInCalendar(Schedule schedule) {
+        return ScheduleInfoSummaryInCalendar.builder()
+                .scheduleId(schedule.getId())
+                .startDateTime(schedule.getStartDateTime())
+                .endDateTime(schedule.getEndDateTime())
+                .hostType(schedule.getHostType().toString())
+                .build();
+    }
+
+    public ScheduleInfoSummariesInCalendar toScheduleInfoSummariesInCalendar(List<ScheduleInfoSummaryInCalendar> schedules) {
+        return ScheduleInfoSummariesInCalendar.builder()
+                .schedules(schedules)
+                .build();
+    }
+
+    public ScheduleInfoSummary toScheduleInfoSummary(Schedule schedule) {
+        return ScheduleInfoSummary.builder()
+                .scheduleId(schedule.getId())
+                .startDateTime(schedule.getStartDateTime())
+                .endDatetime(schedule.getEndDateTime())
+                .title(schedule.getTitle())
+                .hostType(schedule.getHostType())
+                .build();
+    }
+
+    public ScheduleInfoSummaryLists toScheduleInfoSummaries(List<ScheduleInfoSummary> campusSchedules, List<ScheduleInfoSummary> branchSchedules, List<ScheduleInfoSummary> centerSchedules) {
+        return ScheduleInfoSummaryLists.builder()
+                .campusSchedules(campusSchedules)
+                .branchSchedules(branchSchedules)
+                .centerSchedules(centerSchedules)
+                .build();
+    }
+
+
+    public Schedule createScheduleToSchedule(Member member, CreateSchedule request) {
+        return Schedule.builder()
+                .title(request.getTitle())
+                .writer(member)
+                .content(request.getContent())
+                .startDateTime(request.getStartDateTime())
+                .endDateTime(request.getEndDateTime())
+                .semesterPermission(request.getSemesterPermission())
+                .hostType(request.getHostType())
+                .placeSetting(request.getPlaceSetting())
+                .build();
+    }
+
+    public Schedule updateScheduleToSchedule(UpdateSchedule request, Schedule schedule) {
+        schedule.setTitle(request.getTitle());
+        schedule.setContent(request.getContent());
+        schedule.setStartDateTime(request.getStartDateTime());
+        schedule.setEndDateTime(request.getEndDateTime());
+        schedule.setSemesterPermission(request.getSemesterPermission());
+        schedule.setHostType(request.getHostType());
+        schedule.setPlaceSetting(request.getPlaceSetting());
+        return schedule;
+    }
```

Comment:
앞서 코멘트 달았던 것과 같이 엔티티에 Setter 어노테이션 사용을 지양합니다.
따라서, 해당 엔티티 필드들의 수정의 경우 엔티티 내부 클래스에서 메서드를 구현하여 수정하는 게 좋습니다.
```suggestion
    // 엔티티 클래스 내에 다음과 같은 함수처럼 생성하여 수정해주세요.
    public void updateSchedule(UpdateSchedule request) {
        this.title = request.getTitle();
        ~~~
    }
```